Public/js/services/dataService.js

```diff
@@ -0,0 +1,362 @@
+            }]
+        ,
+        school: {
+            schoolId: 0,
+            schoolName: 'Cool School',
+            schoolLocation: '123 Fake St., Nowheresville, KY 40204',
+            schoolDescription: 'A happy little high school where the nights are warm and the ice cream is cool',
+            teachers: [{
+                teacherId: 0,
+                teacherName: 'Stephen Taylor'
+            },
+            {   teacherId: 1,
+                teacherName: 'Marcos Sanchez'
+            }],
+            courses: [{
+                courseId: 0,
+                courseName: 'History of FVII'
+            },
+            {   
+                courseId: 1,
+                courseName: 'People\'s History of the United States'
+            }],
+            students: [0, 1, 2, 3, 4],
+            admins: [{
+                adminId: 0,
+                adminName: 'Steve Young'
+            }]
+        },
+        student: 
+            [
+                {
+                    studentId: 0,
+                    studentFirstName: 'Jordan',
+                    studentLastName: 'Gunther',
+                    studentDOB: 'Tue Jan 19 2016 18:40:27 GMT-0700 (MST)',
+                    assignedPassword: 'JorGun',
+                    teacher: {
+                        teacherId: 0,
+                        teacherName: 'Stephen Taylor'
+                    },
+                    courses: [{
+                        courseId: 0,
+                        courseName: 'History of FFVII',
+                        tests: [{
+                            testId: 0,
+                            testName: 'The Rise of Sephiroth',
+                            completed: false,
+                            grade: 0,
+                            pass: false,
+                            answers: [],
+                            dateTaken: '',
+                            timeTaken: 0,
+                            certificateId: 0
+                        }]
+                    }],
+                    school: {
+                        schoolId: 0,
+                        schoolName: 'Cool School'
+                    }
+                },
+                {
+                    studentId: 1,
+                    studentFirstName: 'Scott',
+                    studentLastName: 'Ogden',
+                    studentDOB: 'Tue Jan 19 2016 18:40:27 GMT-0700 (MST)',
+                    assignedPassword: 'ScoOgd',
+                    teacher: {
+                        teacherId: 0,
+                        teacherName: 'Stephen Taylor'
+                    },
+                    courses: [{
+                        courseId: 0,
+                        courseName: 'History of FFVII',
+                        tests: [{
+                            testId: 0,
+                            testName: 'The Rise of Sephiroth',
+                            completed: true,
+                            grade: 60,
+                            pass: false,
```

Comment:
We can compute "pass or fail" based the number grade, then we can have the teacher decide what constitutes pass or fail for that class or school.